src/components/Hero.jsx

```diff
@@ -3,6 +3,7 @@ import LetsConnect from "./LetsConnect";
 import Lottie from "react-lottie";
 import animationData from "../lotties/person-coding.json";
 import { motion } from "framer-motion";
+import { aboutMe } from "../constants";
```

Comment:
Use one import statement for all variables required from `constants`

Reply:
Here we use just one import to variables from `constants`.
I changed the import statement in the file `Footer.jsx`